client/src/components/helpers/full-width-row.tsx

```diff
@@ -1,16 +1,17 @@
 import { Row, Col } from '@freecodecamp/react-bootstrap';
 import React from 'react';
 
-interface FullWidthRowProps {
+interface FullWidthRowProps extends React.ComponentProps<'div'> {
   children: React.ReactNode;
   className?: string;
 }
 
 const FullWidthRow = ({
   children,
-  className
+  className,
+  ...props
 }: FullWidthRowProps): JSX.Element => (
-  <Row className={className}>
+  <Row className={className} {...props}>
     <Col sm={8} smOffset={2} xs={12}>
```

Comment:
I think it's better to have the props and className as properties of `Col`, because it will make changing the layout of its children easy in tailwind and systems alike, aka you won't need to do `.row .col {cssProperties: value}`

Reply:
But this can be done after we have merged the row and Col components.